podaac/subsetter/dimension_cleanup.py

```diff
@@ -117,8 +117,8 @@ def recreate_pixcore_dimensions(datasets: list):
     dim_dict = {}
     count = 0
     for dataset in datasets:
-        dim_list_shape = list(dataset.dims.values())
-        current_dims = list(dataset.dims.keys())
+        dim_list_shape = list(dataset.sizes.values())
+        current_dims = list(dataset.sizes.keys())
```

Comment:
Can you explain the change here? Is `dims` deprecated in favor of `sizes`?

Reply:
Moslty future warnings i think there will be deprecation in the future but for now its not deprecated yet

/worker/podaac/subsetter/subset.py:531: FutureWarning: The return type of `Dataset.dims` will be changed to return a set of dimension names in future, in order to be more consistent with `DataArray.dims`. To access a mapping from dimension names to lengths, please use `Dataset.sizes`.\r\n  time_vars = list(filter(lambda var_name: 'time' in var_name, dataset.dims.keys()))\r\n